.mdlrc

```diff
@@ -1,2 +0,0 @@
-style '.mdl_style.rb'
-ignore_front_matter true
```

Comment:
> This could break things for people not using pre-commit, or calling `mdl` from their editor.

That's true. I removed this because (as I understand it) these two invocations of mdl are the same:

 - (option 1) `mdl -c .mdlrc`
 - (option 2) `mdl -i -s .mdl_style.rb`

Because the pre-commit configuration, as well as CI, use option 2, neither of them require `.mdlrc`. So, there's some information duplication going on here — suppose we were to ever modify `.mdlrc`, it'd be easier to forget to change the command-line arguments (option 2) wherever they appear.

Actually, maybe a better way is to stop explicitly specifying the command-line arguments (option 2) and just stick to the config file (option 1) everywhere?

Reply:
> Actually, maybe a better way is to stop explicitly specifying the command-line arguments (option 2) and just stick to the config file (option 1) everywhere?

That sounds like a good idea, less typing and duplication.

I _think_ `mdl` looks for `.mdlrc` in the current directory also.

Reply:
> I think `mdl` looks for `.mdlrc` in the current directory also.

Yes, it does; so we don't actually need any argument at all.

(I found it can search upwards in the directory structure too, so it can find `.mdlrc` when called from a subdirectory, but sadly it doesn't then find `.mdl_style.rb`.)

Reply:
Okay, this should be resolved now. I restored `.mdlrc` to its former glory (:)) and `mdl` should pick it up every time it's run (either manually like in CI, or via pre-commit).